client/js/libs/handlebars/index.js

```diff
@@ -0,0 +1,10 @@
+import "./colorClass";
+import "./date";
+import "./diff";
+import "./equal";
+import "./modes";
+import "./parse";
+import "./roundBadgeNumber";
+import "./tojson";
+import "./tz";
+import "./users";
```

Comment:
Well, you could locate the loading file outside of the directory (Ruby-style). But yeah, whatever :-)